tests/spec/unit/templates/cordova/lib/build.spec.js

```diff
@@ -699,6 +699,152 @@ describe('Testing build.js:', () => {
             expect(electronBuilder.userBuildSettings.config.win).toEqual(expectedWin);
         });
 
+        it('should not append package top-level key options if object is empty.', () => {
+            // mock platformConfig, buildConfig and buildOptions Objects
+            const platformConfig = {
+                mac: { package: ['pkg', { }], arch: 'arch', signing: 'signing' }
```

Comment:
```suggestion
                mac: { package: ['pkg', { dmg: { } }], arch: 'arch', signing: 'signing' }
```
The test case is checking the options OF the package, not the object that defines the package. This change is to clarify the test case.